CHANGELOG.md

```diff
@@ -7,8 +7,53 @@ Features in Alpha or Beta stage are tagged as such. We try to follow the same co
 Kubernetes for [feature development
 stages](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions).
 
+Some experimental features can be enabled / disabled using [Feature Gates](docs/feature-gates.md).
+
 ## Unreleased
 
+## 0.8.0 - 2020-07-02
+
+### Added
+
+- Add "Antrea Proxy" implementation to provide Pod-to-Service load-balancing (for ClusterIP Services) directly in the OVS pipeline. ([#772](https://github.com/vmware-tanzu/antrea/pull/772), [@weiqiangt]) [Alpha - Feature Gate: `AntreaProxy`]
```

Comment:
We should mention this fixes NetworkPolicy for Service traffic on Windows, and it is enabled by default on Windows.

Reply:
Done